opentech/apply/funds/templates/funds/includes/activity-feed.html

```diff
@@ -15,7 +15,7 @@ <h4 class="activity-feed__heading">Activity Feed <svg class="icon icon--speech-b
 
     <div class="wrapper wrapper--medium wrapper--activity-feed js-tabs-content">
         <div class="tabs__content tabs__content--current" id="tab-1">
-            {% include "activity/include/comment_list.html" with submission_title=True %}
+            {% include "activity/include/comment_list.html" with submission_title=True editable=False %}
```

Comment:
`editable=False` not necessarily required as will be falsey without this. But happy to include